packages/fx-core/src/common/tools.ts

```diff
@@ -495,3 +495,32 @@ export async function copyFiles(
     });
   }
 }
+
+export function getSiteNameFromResourceId(resourceId: string): string {
+  const result = parseFromResourceId(/providers\/Microsoft.Web\/sites\/([^\/]*)/, resourceId);
```

Comment:
missing `\/` or '$' at the tail of '/providers\/Microsoft.Web\/sites\/([^\/]*)/', so the value maybe incomplete?

Reply:
No. The resource id will be like `subscriptions/1756abc0-3554-4341-8d6a-46674962ea19/resourceGroups/dilinCustomizedRG005/providers/Microsoft.Web/sites/test05abc787-function-webapp` and resource name is the last section without trailing `/`

Reply:
Synced offline and we agree to keep current code.